gpsamson/sandbox
# Update README.md

## What it do?
Updates the Zendesk integration to use the new `/sunrise` endpoint instead of `/cdp`.

For context see:
- https://segment.slack.com/archives/CC97A542H/p1564534125058200
- https://develop.zendesk.com/hc/en-us/community/posts/360033033653-Sunshine-Events-Profiles-deprecation-of-old-API-paths

**Link to Jira Ticket(s)**
https://segment.atlassian.net/browse/DEST-783

**Potential Risks**
None

## Test Plan
- Qualify production data with DOC
- Confirm events are being sent to the new endpoint with a test source
- Confirm with Zendesk that events are hitting the new endpoint

## Rollback Plan
**Pre-merge**
Destinations One-Click will automatically rollback any customer releases if it detects an issue. Manually trigger a release rollback by running this command:
```
robo prod.ssh integrations-doc rollback gps/zendesk
```

**Post-merge**
Revert this PR. The commit will automatically be deployed through autotreb.
